routers/api/v1/repo/release_tags.go

```diff
@@ -55,6 +55,12 @@ func GetReleaseTag(ctx *context.APIContext) {
 		return
 	}
 
+	if err != nil && models.IsErrReleaseNotExist(err) ||
+		release.IsTag || release.RepoID != ctx.Repo.Repository.ID {
```

Comment:
I don't think it's needed to check repoid

Reply:
Sure, i've removed that test